hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/recon/ReconConfigKeys.java

```diff
@@ -68,4 +68,7 @@ private ReconConfigKeys() {
 
   public static final String OZONE_RECON_ADMINISTRATORS_GROUPS =
       "ozone.recon.administrators.groups";
+
+  public static final String OZONE_RECON_TASK_SAFEMODE_WAIT_THRESHOLD
+      = "ozone.recon.task.safemode.wait.threshold";
```

Comment:
Should this be added to `ozone-default.xml`?

If not, at least add a comment describing how this is intended to be used. What the default time unit is (seconds). So advanced users can tune it without digging into the code.

Also it is missing the default value const, which should be `300`.

Reply:
@smengcl - This config is defined as some other task configs in "`ReconTaskConfig`" class. Pls check below code section:
<img width="665" alt="image" src="https://user-images.githubusercontent.com/20607992/209074827-6df12676-4353-46a4-80d7-16bf349a6881.png">

If you want this config to be defined in `ozone-default.xml`, I will do. Default value in above screenshot from `ReconTaskConfig` is 300s only.

Reply:
If we don't want to have this config in the `ozone-default.xml`, we can maybe add it to the [Recon markdown file](https://github.com/apache/ozone/blob/ab608c5d4649b2b320e732b042998bf6ece77cc9/hadoop-hdds/docs/content/concept/Recon.md#L140), at the `Notable configurations` table (like the `ozone.recon.task.missingcontainer.interval`).

Reply:
I see. The doc addition should be good enough. Thanks @devmadhuu and @dombizita .